### PR TITLE
GraphCypherQAChain: Dynamic inputs cypher generation

### PR DESCRIPTION
**Pull request by RafaelXokito**
**Description:** This PR introduces a change to the `cypher_generation_chain` to dynamically concatenate inputs. This improvement aims to streamline the input handling process and make the method more flexible. The change involves updating the arguments dictionary with all elements from the `inputs` dictionary, ensuring that all necessary inputs are dynamically appended. This will ensure that any cypher generation template will not require a new `_call` method patch.

**Issue:** This PR fixes issue #24260.

### CODE REVIEW

File: libs/community/langchain_community/chains/graph_qa/cypher.py [hunk 1]
```diff
@@ -311,12 +311,15 @@ def _call(
         _run_manager = run_manager or CallbackManagerForChainRun.get_noop_manager()
         callbacks = _run_manager.get_child()
         question = inputs[self.input_key]
+        args = {
+            "question": question,
+            "schema": self.graph_schema,
+        }
+        args.update(inputs)
```
[the remark this answers is not in the document]
Comment (RafaelXokito) [reply libs/community/langchain_community/chains/graph_qa/cypher.py:318]: That's a good question, regarding the content duplication, if the input_key for the question is different than "question" it will indeed duplicate because the default template uses "question" and I cant guarantee that the user didn't specified other `cypher_prompt` template.

**Review by eyurtsev — APPROVED** [review]

**Comment by RafaelXokito** [comment]
@eyurtsev should I include tests for more edge cases? I can open another PR.

Give your feedback on this.
